<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_predict</class>
 <widget class="QDialog" name="Dialog_predict">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>49</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>input</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_input">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_input">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_output">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>49</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>output</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_output">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_config">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>49</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>config</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_config">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_gpu">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>49</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>GPU</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_predict</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BT_input</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_predict</receiver>
   <slot>slot_input()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BT_output</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_predict</receiver>
   <slot>slot_output()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BT_config</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_predict</receiver>
   <slot>slot_config()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_predict</receiver>
   <slot>slot_done()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_input()</slot>
  <slot>slot_output()</slot>
  <slot>slot_config()</slot>
  <slot>slot_done()</slot>
 </slots>
</ui>
